pkg/target/manager.go

```diff
@@ -160,6 +205,24 @@ func (m *managerImpl) TargetSeed(ctx context.Context, seedName string) error {
 	})
 }
 
+func (m *managerImpl) DropTargetSeed() (string, error) {
+	currentTarget, err := m.CurrentTarget()
+	if err != nil {
+		fmt.Errorf("failed to get current target: %v", err)
+		return "", nil
+	}
+	targetedName := currentTarget.SeedName()
+	if targetedName != "" {
+		return targetedName, m.patchTarget(func(t *targetImpl) error {
+			t.Seed = ""
+
```

Comment:
I think you also need to drop the targeted shoot, or not?

Reply:
correct, the command should behave similar to the targetSeed cmd